content/product-engineering/planning.md

```diff
@@ -0,0 +1,140 @@
+
+| Week of quarter | Action                                                                                                                                    |
+| --------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| Week 2          | Draft annual vision document for next year<li>VPP/VPE draft annual vision doc                                                             |
+| Week 3          | Gather input from stakeholders<li>VPP/VPE get feedback from exec team and key stakeholders                                                |
+| Week 4          | Update annual vision <li>VPP/VPE update annual vision draft based on feedback                                                             |
+| Week 5          | Share & gather feedback <li>VPP/VPE share annual vision with Product/Eng Directors and gather feedback                                    |
+| Week 6          | Finalize annual vision document <li>VPs update and finalize based on feedback <li>Annual vision added to handbook & communicated on Slack |
+
+### Quarterly planning process
+
+This is the planning process we do each quarter.
+| Week | Action |
+| -------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Week 7 of current quarter<br>5 weeks until next quarter | Gather input from stakeholders on upcoming priorities <li>VPP/VPE gather input from exec team on important problems to solve next quarter <li>Product/Eng Directors gather input from teams on next quarter priorities |
+| Week 8 of current quarter<br>4 weeks until next quarter | Draft department level OKRs <li>VPP/VPE & Product/Eng Directors create first draft of OKRs during [VP team sync](team.md#vp-team-sync)<li>VPP/VPE share with exec team and integrate feedback<li>Product/Eng Directors share with their orgs and integrate feedback |
+| Week 9 of current quarter<br>3 weeks until next quarter | Revise department level OKRs<li>VPP/VPE & Product/Eng Directors discuss and revise OKRs during [VP team sync](team.md#vp-team-sync)<li>VPP/VPE share updates with exec team and integrate feedback<li>Product/Eng Directors share updates with their orgs and integrate feedback |
+| Week 10 of current quarter<br>2 weeks until next quarter | Finalize OKRs<li>VPP/VPE & Product/Eng Directors align on final OKRs during [VP team sync](team.md#vp-team-sync)<li>VPP/VPE get exec team buy in to finalize and approve OKRs<li>OKRs added to the handbook & communicated on Slack |
+| Week 12 of current quarter<br>1 week until next quarter | End of quarter retrospective<li>VPP/VPE & Product/Eng Directors retro previous quarter<li>VPP/VPE and CTO team retro previous quarter & finalize OKR scorecard |
```

Comment:
@jyavorska In week 10 we have a step to add the OKRs to the handbook and communicate them on Slack. We can link to the pages as you suggested to ensure people know exactly where in the handbook it will be updated.

Reply:
I missed that, thanks @msennett22 ! I'll add a suggestion up there instead.